### PR TITLE
Admin mode and lowdb upgrade

### PR DESCRIPTION
**Pull request by tescher**
This combines Nigel's upgrade of lowdb and the admin command mode into one pull request

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -27,20 +53,106 @@ app.post('/', function(req, res, next) {
...
+    }).value()
+
+    if (is_admin) {
+        var commands = message.toLowerCase().split(" ")
+        switch (commands[0]) {
+            case "get":
+                switch (commands[1]) {
+                    case "admin":
+                    case "admins":
+                        return res.send(config.get('admins').value().join())
```
Comment (NigelKibodeaux) [inline index.js]: probably want to join these with a space or a newline character

File: index.js [hunk 2]
```diff
@@ -27,20 +53,106 @@ app.post('/', function(req, res, next) {
...
+            case "set":
+                switch (commands[1]) {
+                    case "admin":
+                    case "admins":
+                        if (commands[2]) {
+                            if (!config.get('admins').find(function(item) {
+                                return item == commands[2]
+                            }).value()) {
+                                config.get('admins').push(
+                                    commands[2]
```
Comment (NigelKibodeaux) [inline index.js]: what format does twilio use? i'm wondering if it would be good to check the format before setting.

File: package.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
 {
-  "name": "plug-at-20-sms",
+  "name": "election-watch-node",
```
Comment (NigelKibodeaux) [inline package.json:2]: oh nice, thanks for fixing. i guess i lazily just copied this from the plug in app and never updated it

File: .gitignore [hunk 4]
```diff
@@ -2,3 +2,4 @@ node_modules
 lastFetch.html
 db.json
```
Comment (NigelKibodeaux) [inline .gitignore:3]: probably want to add `config.json` in here too so it doesn't get accidentally committed

File: message_text.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
     "CONFIRMATION": "Alrighty, you're signed up. Send STOP if you want the messages to stop and START if you change your mind after stopping.",
     "INSTRUCTIONS": "Sorry, I don't know what you want. Send STOP if you want the messages to stop and START if you change your mind after stopping.",
-    "NOTIFICATION": "The page has been updated. Refresh the page after following the link: http://results.elections.alaska.gov/data/results.htm",
-    "GOODBYE": "You've been unsubscribed. Later!"
+    "NOTIFICATION": "The page has been updated. Refresh the page after following the link: ",
+    "GOODBYE": "You've been unsubscribed. Later!",
+    "ADMIN-HELP": "GET ADMINS, SET ADMIN <phone>, REMOVE ADMIN <phone>, GET URL, SET URL <url>, REMOVE SUBSCRIBERS"
```
Comment (NigelKibodeaux) [inline message_text.json]: `ADMIN-HELP` should be `ADMIN_HELP` so that `res.send(text.ADMIN_HELP)` works

**Review by NigelKibodeaux — CHANGES_REQUESTED** [review]
Looks great! Cool idea too! I found one error and had a few other thoughts/suggestions.

**Review by NigelKibodeaux — APPROVED** [review]

File: message_text.json (hunk 5 above)
Comment (tescher) [reply message_text.json]: Good catch. Can't test it because Twilio intercepts HELP currently

File: .gitignore (hunk 4 above)
Comment (tescher) [reply .gitignore:3]: Done

File: index.js (hunk 2 above)
Comment (tescher) [reply index.js]: Forced everything to +1NNNNNNNNNN. Regex is so much fun!

**Pull request merged**